funtests/src/test/scala/me/rexim/morganey/funtests/FibFuntest.scala

```diff
@@ -0,0 +1,14 @@
+package me.rexim.morganey.funtests
+
+import java.io.File
+
+import org.scalatest._
+import scala.sys.process._
+
+class FibFuntest extends FlatSpec with Matchers {
+  "15 first fibonacci numbers sample" should "print 15 first fibonacci numbers" in {
+    val expectedOutput = "numbers: [0,1,1,2,3,5,8,13,21,34,55,89,144,233,377]\n"
+    val actualOutput = "java -jar ./target/scala-2.11/morganey.jar ./docs/samples/fib.mgn".!!
```

Comment:
I recommend to factor this out into a function or a constant, to remove the duplication of `"java -jar ./target/scala-2.11/morganey.jar"` everywhere.